CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## XX.XX.X
 * Added 'UnityWebRequest' as the networking handler for WebGL builds.
```

Comment:
additions above, deprecations below https://countly.getoutline.com/doc/sdk-standards-and-processes-qSfQbSoQsX#:~:text=be%20the%20following%3A-,!!%20Major%20breaking%20change%20!!,-!%20Minor%20breaking%20change